share/spice/just_delete_me/content.handlebars

```diff
@@ -0,0 +1,15 @@
+{{#if notes}}
+  <p>
+    {{notes}}
+  </p>
+{{/if}}
+{{#if delete_url}}
+  <p>
+    <b>URL:</b> <a href='{{delete_url}}'>{{delete_url}}</a>
```

Comment:
Please remove the `<b>` -- we don't typically bold text. If anything we use different colours